CI: Update SHA Pins ci-docker

## Maintainer Summary
2026-07-21 00:09:44 UTC

## CI Log Error
https://github.com/InternationalColorConsortium/iccDEV/actions/runs/29787506185/job/88502177744#step:8:2252
Error: buildx failed

## Research
2026-07-21 00:15:25 UTC

Research: There is a new constructor parameter named imageData in iccProfileVisualize.cpp

### Log
```
#16 88.83 [ 80%] Building CXX object Tools/IccProfileVisualize/CMakeFiles/iccProfileVisualize.dir/opt/iccdev/Tools/CmdLine/IccProfileVisualize/iccProfileVisualize.cpp.o
#16 89.49 /opt/iccdev/Tools/CmdLine/IccProfileVisualize/iccProfileVisualize.cpp: In constructor 'imageData::imageData(const std::string&, void*, int, int, int, int, int, bool)':
#16 89.49 /opt/iccdev/Tools/CmdLine/IccProfileVisualize/iccProfileVisualize.cpp:386:44: error: declaration of 'imageData' shadows a member of 'imageData' [-Werror=shadow]
#16 89.49   386 |   imageData(const std::string &name, void *imageData, int imageWidth, int imageHeight,
#16 89.49       |                                      ~~~~~~^~~~~~~~~
#16 89.49 /opt/iccdev/Tools/CmdLine/IccProfileVisualize/iccProfileVisualize.cpp:380:40: note: shadowed declaration is here
#16 89.49   380 | struct imageData : dataAbstractionBase {
#16 89.49       |     
```

--- a/Tools/CmdLine/IccProfileVisualize/iccProfileVisualize.cpp
+++ b/Tools/CmdLine/IccProfileVisualize/iccProfileVisualize.cpp
@@ -377,13 +377,16 @@ enum {
     IMAGE_MODE_CMYK = TIFF_MODE_CMYK,
 };
 
+// this takes ownership of the buffer pointer passed in
+// pointer must be allocated with new[]
+// TODO - can I use a unique_ptr to manage this?
 struct imageData : dataAbstractionBase {
 
   imageData() : width(0), height(0), channels(0), depth(0),
                 mode(IMAGE_MODE_GRAY_BLACKZERO),
                 isFloatingPoint(false), data(NULL) {}
 
-  imageData(const std::string &name, void *imageBuffer, int imageWidth, int imageHeight,
+  imageData(const std::string &name, uint8_t *imageBuffer, int imageWidth, int imageHeight,
             int imageChannels, int imageDepth, int imageMode,
             bool dataIsFloatingPoint = false ) {
     object_name = name;
@@ -393,7 +396,7 @@ struct imageData : dataAbstractionBase {
     depth = imageDepth;
     mode = imageMode;
     isFloatingPoint = dataIsFloatingPoint;
-    data = (uint8_t*)imageBuffer;
+    data = imageBuffer;
   }
 
   ~imageData() {

--- a/Tools/CmdLine/IccProfileVisualize/iccProfileVisualize.cpp
+++ b/Tools/CmdLine/IccProfileVisualize/iccProfileVisualize.cpp
@@ -383,7 +383,7 @@ struct imageData : dataAbstractionBase {
                 mode(IMAGE_MODE_GRAY_BLACKZERO),
                 isFloatingPoint(false), data(NULL) {}
 
-  imageData(const std::string &name, void *imageData, int imageWidth, int imageHeight,
+  imageData(const std::string &name, void *imageBuffer, int imageWidth, int imageHeight,
             int imageChannels, int imageDepth, int imageMode,
             bool dataIsFloatingPoint = false ) {
     object_name = name;
@@ -393,7 +393,7 @@ struct imageData : dataAbstractionBase {
     depth = imageDepth;
     mode = imageMode;
     isFloatingPoint = dataIsFloatingPoint;
-    data = (uint8_t*)imageData;
+    data = (uint8_t*)imageBuffer;
   }
 
   ~imageData() {

